variables.tf

```diff
@@ -200,3 +206,42 @@ variable "zone_dns_name" {
     error_message = "The DNS name cannot end with '.local'."
   }
 }
+
+variable "azs" {
+  description = "value"
+  type        = list(string)
+}
+
+variable "measure_latency" {
```

Comment:
Add prefixes to the variables to note where they are used, for example this could be `route53_measure_latency` while `sns_topic_endpoint` could become `monitoring_sns_topic_endpoint`